src/journal/journalctl.c

```diff
@@ -267,6 +268,8 @@ static int help(void) {
                "     --no-tail               Show all lines, even in follow mode\n"
                "     --truncate-newline      Truncate entries by first newline character\n"
                "  -q --quiet                 Do not show info messages and privilege warning\n"
+               "     --synchronize-on-exit=BOOL\n"
+               "                             Wait for Journal synchronization before exiting\n"
```

Comment:
I don't think it makes sense to make this a bool. It'd only be ever explicitly enabled in specific circumstances and never disabled. And it doesn't match the style of the other options.

Reply:
dunno, i think these days it's nicer to use switches with args for things that are more in the realm of "flag indicating a minor change to a primary operation", so that one clearly turn it either on or off. But stuff that is more in the realm of "request some primary operation" or other more central option should get a switch without bool param.

dunno, to some degree this is bikeshedding, no?